params/hooks_libevm.go

```diff
@@ -192,6 +196,6 @@ func (b *BlockContext) Timestamp() uint64 {
 	return b.time
 }
 
-func (b *BlockContext) GetPredicateResultsBytes() []byte {
-	return b.predicateResultsBytes
+func (b *BlockContext) GetPredicateResults(txHash common.Hash, precompileAddress common.Address) []byte {
```

Comment:
should this return `predicate.Results`?

Reply:
or keep the byte in name?

Reply:
This is how it is in master so I left it like that for now.